client/js/lounge.js

```diff
@@ -391,6 +389,15 @@ $(function() {
 	}
 
 	function renderNetworks(data) {
+		data.networks.forEach(function(network) {
```

Comment:
`splice` could be better fit for this. It returns spliced elements. `slice` copies the array.